<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiTermWin</class>
 <widget class="QDialog" name="multiTermWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分屏终端</string>
  </property>
  <widget class="QWidget" name="termWidget4" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>400</y>
     <width>581</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="termWidget2" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>611</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="termWidget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="termWidget3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>641</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="send2allBox">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入到所有会话</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
